src/compressor.js

```diff
@@ -0,0 +1,146 @@
+define(function (require) {
+	'use strict';
+
+	var p5sound = require('master');
+	var Effect = require('effect');
+  var CustomError = require('errorHandler');
+
+
+  /**
+   * Built on Web Audio Dynamics Compressor Node
+   * https://www.w3.org/TR/webaudio/#the-dynamicscompressornode-interface
+   *
+   * @class p5.Compressor
+   * @return {Object} Compressor Object
+   */
+	p5.Compressor = function() {
```

Comment:
I don't have a problem with this! I didn't include `set` parameters just to match what seemed to be standard in other effects